packages/security-cordova/plugin.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='utf-8'?>
+<plugin
+    id="cordova-plugin-aerogear-device-trust"
```

Comment:
I don't know if this is a requirement but I think we should name plugin and package the same.

Reply:
Nice catch, thanks